symplyphysics/laws/thermodynamics/work_is_volume_integral_of_pressure.py

```diff
@@ -0,0 +1,81 @@
+from sympy import Eq, Integral, Point2D, solve
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.core.geometry.line import two_point_function
+from symplyphysics.laws.thermodynamics import infinitesimal_work_in_quasistatic_process as work_law
+
+# Description
+## The pressure-volume work occurs when the volume of a system changes.
+
+# Law: W = Integral(p(V), V)
+## W - work done by gas
+## p - gas pressure
+## V - gas volume
+
+# Conditions
+## - The process is reversible or quasi-static.
+## - The system is closed.
+
+work = Symbol("work", units.energy)
+pressure = Function("pressure", units.pressure)
+volume = Symbol("volume", units.volume)
+volume_before = Symbol("volume_before", units.volume)
+volume_after = Symbol("volume_after", units.volume)
+
+law = Eq(work, Integral(pressure(volume), (volume, volume_before, volume_after)))
+
+# Derive law from infinitesimal counterpart
+
+# Prepare law for integration
+_infinitesimal_law = work_law.law.subs({
+    work_law.infinitesimal_work_done: 1,
+    work_law.infinitesimal_volume_change: 1,
+    work_law.pressure_inside_system: pressure(volume),
+})
+
+_integrated_law = Eq(
+    _infinitesimal_law.lhs.integrate(work),
+    _infinitesimal_law.rhs.integrate((volume, volume_before, volume_after))
```

Comment:
I'm still not satisfied with this proof. For example we can change RHS integration limits, eg `(volume, volume_before, volume_after / 2)` yet the total work won't change. It means that left and right hand side integrations are not linked to each other.

Reply:
Then the best option would be to have `_work(volume_before) = 0, _work(volume_after) = work`